src/MicrosoftTeams.schema.json

```diff
@@ -247,38 +248,18 @@
                                         "properties": {
                                             "title": {
                                                 "type": "string",
-                                                "description": "The one word command name",
-                                                "maxLength": 16
-                                            },
-                                            "usageHint": {
-                                                "type": "string",
-                                                "description": "A short usage template for the command",
-                                                "maxLength": 32
-                                            },
-                                            "shortDescription": {
-                                                "type": "string",
-                                                "description": "A short description of the command",
+                                                "description": "The bot command name",
                                                 "maxLength": 32
```

Comment:
Are you sure that 32 characters is right for the maxLength here? Will the input menu in our UI actually be able to fit a command name this long?